tools/count_citation.py

```diff
@@ -4,16 +4,19 @@
 from pathlib import Path
 
 import pandas as pd
-import plotly.express as px
 import requests
 from bids_website.utils import data_dir
 from pyzotero import zotero
 from rich import print
 
 DEBUG = False
-VERBOSE = False
+VERBOSE = True
 MINIMUM_YEAR = 2016
 
+TOKEN_FILE = "/home/remi/Documents/tokens/open_citations_access_token.txt"
```

Comment:
```suggestion
# TODO: switch this to a .env file
os.environ["OPEN_CITATIONS_ACCESS_TOKEN"] = "/home/remi/Documents/tokens/open_citations_access_token.txt"
TOKEN_FILE = os.environ.get("OPEN_CITATIONS_ACCESS_TOKEN", "")
```

Reply:
We will have to revisit anyway because of the massive discrepancies in numbers that @ericearl noted depending on the data source we are using for counting citations. 

Plus I have an extension in vscode that ALWAYS show me all the TODOs in my codebase... 😉